analysis/active_analyses.R

```diff
@@ -520,9 +531,9 @@ df$name <- paste0("cohort_",df$cohort, "-",
 
 # Select certain models --------------------------------------------------------
 
-# df <- df[df$analysis == "main" | df$analysis == "sub_covid_hospitalised" | df$analysis == "sub_covid_nonhospitalised",]
+df <- df[df$analysis == "main" | df$analysis == "sub_covid_hospitalised" | df$analysis == "sub_covid_nonhospitalised",]
 
```

Comment:
Now we have around 300 analysis (maybe less) so you could add a line(s) to include all the analysis that will be run for the composite ai).